programs/native/bpf_loader/Cargo.toml

```diff
@@ -6,19 +6,25 @@ authors = ["Solana Maintainers <maintainers@solana.com>"]
 repository = "https://github.com/solana-labs/solana"
 license = "Apache-2.0"
 
+[features]
+bpf_c = []
+
 [dependencies]
 bincode = "1.0.0"
 byteorder = "1.2.1"
 elf = "0.0.10"
 env_logger = "0.5.12"
 libc = "0.2.43"
 log = "0.4.2"
-solana_rbpf = "=0.1.3"
+solana_rbpf = "=0.1.4"
 serde = "1.0.27"
 serde_derive = "1.0.27"
 solana-sdk = { path = "../../../sdk", version = "0.11.0" }
 
 [lib]
 name = "solana_bpf_loader"
-crate-type = ["cdylib"]
+crate-type = ["lib", "cdylib"]
+
+[[bench]]
+name = "bpf_loader"
```

Comment:
Do we run this bench in CI somewhere?

Reply:
Not yet, plan to run both this bench and the bpf_c featured tests